packages/react-components/lib/dispensers/dispenser-accordion.tsx

```diff
@@ -91,20 +102,18 @@ export const DispenserAccordion = React.forwardRef(
             variant: statusLabelClass ? 'normal' : 'unknown',
           }}
         />
-        {dispenserState ? (
-          <ItemAccordionDetails>
-            <DispenserInfo dispenser={dispenserState} />
-          </ItemAccordionDetails>
-        ) : (
-          <React.Fragment>
-            <Divider />
-            <Typography className={classes.typography} variant="body1">
-              The state of <b>{dispenser}</b> dispenser is unknown. Please check if{' '}
-              <b>{dispenser} </b>
-              is working properly.
-            </Typography>
-          </React.Fragment>
-        )}
+        <ItemAccordionDetails>
+          <ItemUnknown
+            errorMsg={'Dispenser is not sending states. Please check if it is working properly.'}
+            showError={!dispenserState}
```

Comment:
Should we simplify the props by removing `showError` and use `errorMsg` to test if we should enable the blur?